docs/eigenda/deployed-contracts.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 8
+title: Contracts
+---
+
+## EigenDA Contracts
+
+For more information on the contracts and their technical specifications, please see:
+[EigenLayer Middleware Docs](https://github.com/Layr-Labs/eigenlayer-middleware/tree/m2-mainnet/docs).
+
+### Current Mainnet Deployment
+
+No contracts have been deployed to mainnet yet.
+
+### Current Testnet Deployment
+
+| Name                           | Solidity                                                                                                   | Proxy                                                          | Implementation                                                | Notes                                                                                                            |
+|--------------------------------|------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------|---------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| RegistryCoordinator            | [`BLSRegistryCoordinatorWithIndices.sol`](https://github.com/Layr-Labs/eigenlayer-middleware/blob/v0.1.0-m2-goerli/src/BLSRegistryCoordinatorWithIndices.sol) | [`0x7661dA87296c0A245d11ADeD7f48265463bc3eC6`](https://goerli.etherscan.io/address/0x7661dA87296c0A245d11ADeD7f48265463bc3eC6) | [`0x7dfE0C25483E420eccD04eab8Be44c75a0CE82B5`](https://goerli.etherscan.io/address/0x7dfE0C25483E420eccD04eab8Be44c75a0CE82B5) | Proxy: [OpenZeppelin TUP@4.7.1](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.7.1/contracts/proxy/transparent/TransparentUpgradeableProxy.sol) |
```

Comment:
Looks like the links to solidity files are from `v0.1.0-m2-goerli` branch.
The branch we deployed yesterday was from `m2-mainnet-fixes`